librz/include/rz_debug.h

```diff
@@ -358,11 +359,12 @@ typedef struct rz_debug_plugin_t {
 	const char *license;
 	const char *author;
 	const char *version;
-	//const char **archs; // MUST BE DEPRECATED!!!!
 	ut32 bits;
 	const char *arch;
 	int canstep;
 	int keepio;
+	int (*init)(RzDebug *dbg, void **user);
```

Comment:
Maybe add typedef func type?

Reply:
What do you mean exactly? All the other plugin callbacks are defined in-place too so I am not sure why we would want to use a typedefd one here.

Reply:
please use bool and not int

Reply:
changed to bool